### PR TITLE
update zh_CN translation

### PR DESCRIPTION
**Pull request by bigQY**
Added Chinese translation for some texts.

### CODE REVIEW

File: .github/workflows/windows-qt.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 name: Windows-Qt
 
 on:
+  workflow_dispatch:
```
Comment (abouvier) [inline .github/workflows/windows-qt.yml:7]: Why did you add that?

Comment (bigQY) [reply .github/workflows/windows-qt.yml:7]: I added this line of code to be able to run workflow manually, which is not necessary in this pr. I chose the wrong branch when I submitted this pr, I will close the current pr and submit a new one. #699 

Thank you.
